requirements_bjrnn.txt

```diff
@@ -1,9 +0,0 @@
-matplotlib~=3.3.4
```

Comment:
I think this file should still be included, as the BJ-RNN baseline only works with the very old PyTorch version (that doesn't have internal counter checks) as indicated here.